themes/app/src/js/app.js

```diff
@@ -4,6 +4,6 @@ import CustomNav from './components/CustomNav.vue';
 new Vue({
     el: '#app',
     components: {
-        'custom-nav': CustomNav,
+        'customnav': CustomNav,
```

Comment:
hmm.. I thought we were renaming this to `nav` ?

Reply:
It doesn't work, hence I renamed it.